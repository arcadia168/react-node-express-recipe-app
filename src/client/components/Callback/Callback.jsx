import React, { Component } from 'react';

class Callback extends Component {
  render() {
    // const style = //...
    const style = undefined;
    return (
      <div style={style}>
        LOADING
      </div>
    );
  }
}

module.exports = Callback;