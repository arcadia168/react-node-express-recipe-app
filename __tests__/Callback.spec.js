import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Callback from 'Callback';

describe('Callback', () => {
    it('renders correctly', () => {
        const tree = renderer
          .create(<Callback />)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
});