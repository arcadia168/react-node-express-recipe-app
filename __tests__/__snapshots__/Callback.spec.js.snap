// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Callback renders correctly 1`] = `
<div
  style={undefined}
>
  LOADING
</div>
`;
